app/assets/javascripts/discourse/app/components/emoji-picker.js

```diff
@@ -354,6 +360,13 @@ export default Component.extend({
     return document.activeElement.closest(item) ? document.activeElement : null;
   },
 
+  _applyAccessibility() {
+    const allEmojis = document.querySelectorAll(this.elements.allEmojis);
```

Comment:
Ah, since you're adding support to `performEmojiUnescape` for the `tabIndex option, I don't think we need the approach via the `didInsertElement` hook here. 

Instead, we can update the entries in emoji-group-sections.hbs to look like this: 

```
{{replace-emoji ":grinning:" (hash lazy=true tabIndex="0")}}
```

Note that it needs to be done in the rake task that generates that special emoji-group-sections handlebars file, see note at the start of the file.